craft_archives/repo/package_repository.py

```diff
@@ -236,8 +272,31 @@ def unmarshal_package_repositories(
 class PackageRepositoryAptPPA(PackageRepository):
     """A PPA package repository."""
 
-    ppa: str
-    key_id: KeyIdStr | None = Field(default=None, alias="key-id")
+    ppa: str = Field(
+        description="The short name for the PPA.",
+        examples=["mozillateam/firefox-next"],
+    )
+
+    key_id: KeyIdStr | None = Field(
+        default=None,
+        alias="key-id",
+        description="The GPG identifier of the repository.",
+        examples=["590CA3D8E4826565BE3200526A634116E00F4C82"],
+    )
+    """The GPG identifier of the repository.
+
+    The key is also known as a long-form thumbprint or fingerprint.
```

Comment:
We should be careful to separate project file keys from other kinds of keys.

```suggestion
    A GPG key is also known as a long-form thumbprint or fingerprint.
```

Reply:
https://github.com/canonical/craft-archives/pull/162/commits/540a69b84954ead70d4c21f736d3a6a484d55f1e